src/bramble/client/ProjectStats.js

```diff
@@ -1,154 +1,259 @@
 define([
     // Change this to filer vs. filer.min if you need to debug Filer
-    "thirdparty/filer/dist/filer"
+    "thirdparty/filer/dist/filer.min",
 ],function(Filer){
     "use strict";
 
-    function ProjectStats() {};
+    var ProjectStats = {};
```

Comment:
This doesn't seem right - this is an Object definition, we can't just rewrite that to a singleton object.